### DISCUSSION TITLE
New MISSION.MD - please read

### DISCUSSION THREAD

**Discussion by daveshap** [daveshap/ACE_Framework#146] [Announcements]
I've spoken with the leadership team and realized that there needed to be some clarity around expectations, leadership, mission, goals, and so on. Accordingly, I've added a `mission.md` to bot the ACE and HAAS repos. Linked below. While everyone is here voluntarily, my hope is that everyone will read these mission statements. They are less than 3000 words (so about the chapter of a book). Understand the mission, method, and ethos. Discuss them. Share them. Embody them. 

Some highlights from the insights I gained from discussing with the core team:

- The mission and methodology wasn't really clear, are we trying to build a useful product or conduct science. The answer is unequivocally science - we are pushing the boundaries of what is possible. 
- What does the goal look like? How do we know what we're building and testing? I've clarified what I mean by "full autonomy" and defined it more clearly. But the goal is quite simple: create fully autonomous, robust, and safe machines in all their forms. 
- How do we get there and how do we know we are succeeding? I've outlined the methodology that I've used over the last four years to get where I am, as well as my attitude towards failure (e.g. "There is no such thing as failure, only learning") In point of fact *we should aim to fail.* If we aren't failing, we aren't pushing hard enough. 

There is quite a lot more, but I wanted to prime the pump and pique your curiosity. 

- https://github.com/daveshap/ACE_Framework/blob/main/mission.md
- https://github.com/daveshap/OpenAI_Agent_Swarm/blob/main/mission.md

Both of these mission documents are identical, but they are conveniently located within each repo.

**Comment by zmef** [comment]
"In point of fact we should aim to fail"
my life plan
